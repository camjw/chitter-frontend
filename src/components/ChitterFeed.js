import React from 'react'

export default class ChitterFeed extends React.Component {
  render() {
    console.log(this.props.peeps)
    return(
      <ul>
        {this.props.peeps.map((peep, index) => (
          <div key={index}>
              {peep.body} peeped by {peep.user.handle}
          </div>
        ))}
      </ul>
    )
  }
}
