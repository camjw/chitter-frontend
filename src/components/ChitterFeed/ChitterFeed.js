import React from 'react';
import Peep from '../Peep';

export default class ChitterFeed extends React.Component {
  render() {
    return (
      <ul>
        <hr />
        {this.props.peeps.map((peep, index) => (
          <div key={index}>
            <Peep peep={peep} />
          </div>
        ))}
      </ul>
    );
  }
}
