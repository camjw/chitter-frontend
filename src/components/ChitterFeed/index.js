import ChitterFeed from './ChitterFeed'

export default ChitterFeed
