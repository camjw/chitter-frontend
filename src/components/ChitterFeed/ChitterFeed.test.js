import React from 'react';
import { shallow } from 'enzyme'
import ChitterFeed from './ChitterFeed'

const chitterFeedWrapper = shallow(<ChitterFeed peeps={ [{ body: 'Hello test', user: { handle: 'Hello' } }] }/>)

describe('ChitterFeed', () => {
  describe('#render', () => {
    it('renders some HTML', () => {
      expect(chitterFeedWrapper.html()).toMatchSnapshot()
    })
  })
})
