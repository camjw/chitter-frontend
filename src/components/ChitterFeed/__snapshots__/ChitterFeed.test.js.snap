// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChitterFeed #render renders some HTML 1`] = `"<ul><hr/><div><div class=\\"container\\"><div class=\\"row\\">Hello test</div><div class=\\"row\\">Peeped by Hello a few seconds ago.</div><hr/><br/></div></div></ul>"`;
